julia/utils.jl

```diff
@@ -8,4 +8,25 @@ end
 
 function getTime()::Integer
     return round(Integer, 1e3*(time()-1.6e9))
+end
+
+# Check for errors before they happen
+function testConfiguration()
```

Comment:
This only requires constants that are defined in constants.jl to work and is a utilities function